ChangeLog.org

```diff
@@ -1,5 +1,44 @@
 * Changelog for Juvix
 
+** 0.1.1.0
+ - _Summary_:
+   + new syntax for effect handlers in Juvix.
+
+ - _Main contributions_:
+   + adds `effect` and `handler` syntaxes (also `_ via _`);
+   + removes do notation as `Prelude.»=`.
+
+*** Added Syntaxes
+#+BEGIN_SRC ocaml
+effect Print where
+  | op print : string -> ()
+  | return   : x -> string
+#+END_SRC
+A new effect Print is defined, and its interface signature. All
+effects must have a `return` clause, and hopefully operation
+signatures as well (an effect without operations is just a pure
+computation).
+
+#+BEGIN_SRC ocaml
+handler print where
+  | op print  = %IO.print
+  | return x  = toString x
+#+END_SRC
+A handler provides an implementation of at least one effect and
+a return clause, and it can implement as many operations as the
+user wants.
+
+#+BEGIN_SRC ocaml
+let main = prog via print
```

Comment:
The `print` here refers to handler named `print` and not to the `print` operation, right?

Reply:
You're correct, I'm gonna change the name of the handler to `printer` so it won't be confusing anymore. Thanks.